daisy_workflows/image_build/kokoro/kokoro_image_preparer.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+# Creating a custom kokoro image from go/kokoro-custom-prod-linux-vm
+
+metadata_url="http://metadata.google.internal/computeMetadata"
+
+GetMetadataAttribute() {
+    attribute="$1"
+
+    echo "Fetching ${attribute}"
+    url="$metadata_url/v1/instance/attributes/$attribute"
+    attribute_value=$(curl -H "Metadata-Flavor: Google" -X GET "$url")
+}
+
+# Must have a "kbuilder" user
+useradd -c "kbuilder user" -m -s /bin/bash kbuilder
+groupadd kokoro
+usermod -aG kokoro kbuilder
+
+# kbuilder must allow login with ssh public/private key authentication
+GetMetadataAttribute "kokoro_authorized_keys" && authorized_keys="$attribute_value"
+mkdir ~/.ssh
```

Comment:
it's better to ignore failure when making dir

Reply:
Sorry, didn't see this. Why is that? This directory has to be made for this image to work.